frontend/components/ArticleBody.tsx

```diff
@@ -20,6 +20,7 @@ import {
     tablet,
 } from '@guardian/pasteup/breakpoints';
 import { pillarMap, pillarPalette } from '../pillars';
+import { ArticleRenderer } from './elements/ElementRenderer';
```

Comment:
It's the convention throughout the code to use the same name for the export as the file so I'd say let's rename the named export`ArticleRenderer` to `ElementRenderer`.

Reply:
The original plan for this file was to export both `ArticleRenderer` and `ElementRenderer`. This wasn't a good idea, so I'll rename the file.